app/src/main/java/com/hamdan/forzenbook/domain/usecase/login/LoginGetTokenUseCase.kt

```diff
@@ -1,7 +1,5 @@
 package com.hamdan.forzenbook.domain.usecase.login
 
-import com.hamdan.forzenbook.data.repository.LoginData
-
 interface LoginGetTokenUseCase {
-    suspend operator fun invoke(email: String, code: String): LoginData?
+    suspend operator fun invoke(email: String, code: String): String
```

Comment:
The UI or VM doesn't need a String. Why return String? Who to?

Reply:
fair, adjusted by removing the return altogether. added improved commenting for further implementation.